### PR TITLE
Feature/packages/blog title #cmx 135

### PR DESCRIPTION
**Pull request by Mar0cas**
Created component and all necessary logic to be functional. Updated Section Component, so we can use `is_post` argument.

### CODE REVIEW

File: packages/organisms/blog-title/src/component/Defaults.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { DefaultProps } from './Types';
+
+const Defaults: DefaultProps = {
+  title: 'lorem Ipsum',
```
Comment (ricardo-valente) [inline packages/organisms/blog-title/src/component/Defaults.ts:4]: why title has a default value?

File: packages/organisms/blog-title/src/__tests__/__snapshots__/BlogTitle.test.tsx.snap [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  data-layout="blog_title"
+>
+  <div
+    class="container"
+  >
+    <div
+      class="row"
+    >
+      <div
+        class="wrapper wrapper--full wrapper--darkblue -primary -shadow"
```
Comment (ricardo-valente) [inline packages/organisms/blog-title/src/__tests__/__snapshots__/BlogTitle.test.tsx.snap:15]: `wrapper--full` does not exists, only big or normal

File: packages/organisms/blog-title/src/component/Defaults.ts (hunk 1 above)
Comment (Mar0cas) [reply packages/organisms/blog-title/src/component/Defaults.ts]: Removed

File: packages/organisms/blog-title/src/__tests__/__snapshots__/BlogTitle.test.tsx.snap (hunk 2 above)
Comment (Mar0cas) [reply packages/organisms/blog-title/src/__tests__/__snapshots__/BlogTitle.test.tsx.snap:15]: Fixed

**Pull request merged**